src/web/GestureView.tsx

```diff
@@ -188,7 +188,7 @@ export class GestureView extends RX.ViewBase<Types.GestureViewProps, {}> {
     }
 
     private _getPanPixelThreshold = () => {
-        return this.props.panPixelThreshold > 0 ? this.props.panPixelThreshold : _panPixelThreshold;
+        return (this.props.panPixelThreshold && this.props.panPixelThreshold > 0) ? this.props.panPixelThreshold : _panPixelThreshold;
```

Comment:
Zero values work OK in this logic, but it would be better if the check for undefined was more explicit.